app/models/agents/post_agent.rb

```diff
@@ -30,6 +30,9 @@ class PostAgent < Agent
         If `emit_events` is set to `true`, the server response will be emitted as an Event and can be fed to a WebsiteAgent for parsing (using its `data_from_event` and `type` options). No data processing
         will be attempted by this Agent, so the Event's "body" value will always be raw text.
         The Event will also have a "headers" hash and a "status" integer value.
+        
+        If `output_mode` is set to `merge`, the original contents of the received Event will be merged into the emitted Event.
```

Comment:
Actually it is the other way around :). The emitted Event is merged into the original event.